packages/zowe-explorer/__tests__/__unit__/dataset/DatasetTree.unit.test.ts

```diff
@@ -2412,3 +2434,112 @@ describe("Dataset Tree Unit Tests - Function rename", () => {
         await testValidDsName("INVALID-DATASET-NAME");
     });
 });
+
+describe("Dataset Tree Unit Tests - Function checkFilterPattern", () => {
```

Comment:
I'm curious if we should try to simplify the `checkFilterPattern` function.
I'm sure ChatGPT may have some suggestions :wink:
(I already tried and it condensed the function down to 5 lines) :yum: